.github/workflows/build.yaml

```diff
@@ -10,35 +10,28 @@ on:
     branches:
       - master
 
+# When updating this version, also update the version in
+# flutter-version.txt.
+#
+# This version should be updated periodically and should generally
+# track the latest Flutter dev release. Dev releases come out ~weekly;
+# the versions can be viewed at https://github.com/flutter/flutter/tags.
 env:
-  CHANNEL: dev
+  CHANNEL: 2.4.0-4.1.pre
```

Comment:
(There's a `2.5.0-1.0.pre` dev now, not sure if it's worth bumping again before landing?)